### DISCUSSION TITLE
expansive talent rant

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#8609] [Unstable feedback]
i hope you like reading

**Captain**
**Gunslinger tree**
This is the tree captains will (or should) get when they want to take an active role outside the submarine, or when they want to be scarier to the crew.

**Once Upon A Time On Europa**
This talent is balanced

**Quickdraw** and **Tumble**
Honestly i think this is a fair choice you have to pick, while quickdraw grants you an immediate 40%+ for your next shot, tumble allows you to shoot 2 shots at the same time if you're using akimbo revolvers with 40%+ damage for each but only if you get stunned.

**Drunken Sailor** and **Sailor With No Name**
Here drunken sailor has the clear advantage, while sailor with no name gives you the ability to save 1000 marks on healing reaper's tax, the chem addiction and 20+ medical skill does not make you better than a medic. Drunken sailor allows you to not only turn the captain into an efficient combat class with just 1 cheap item but also farm skill for the entire crew like no other; I suggest limiting the amount of skill a captain with drunken sailor can give in one game, and making the cigar from sailor with no name give you a damage bonus affliction for handguns, to keep the theme of the talent tree. If you add this it will mean you can either pick to do even more damage or have some kind of ethanol based armor, making it another combat oriented talent choice.

**Bounty Hunter**
The whole bounty feature of the talent is fine, it rewards killing enemies you would usually avoid.
Now the handcannon is where i have a problem personally, first of all it's FAR weaker than just using 2 default revolvers with default ammo. The talents that are supposed to make the handcannon better don't help much either since the damage is still not enough to 1shot a mudraptor  even if you hit the jaw:
https://user-images.githubusercontent.com/55637865/159947730-5f2e392a-e502-4034-bf41-d0feaf505334.mp4

I have 2 suggestions to change the handcannon:
Bring back the 80 damage
or
Give it 100% penetration.

**Skipper tree**
This is usually the first tree everyone gets just so they can get the other trees faster, I assume that is intended

**Traveling Tradesman** and **Evasive Maneuvers**
Traveling Tradesmanis alright, it benefits those who like to take their time to control Europa, i would just reduce the reward for each outpost a bit, maybe 300mk.
Now Evasive Maneuvers is problematic, the 75+ helm skill when the hull is damaged is not quite useful since it lasts so little and it becomes straight up useless after the player reaches 100 helm skill, the 15%+ reward is fine, tho but i don't think it's enough to make up for the rather dissappointing side of the talent.

**Down With The Ship** and **Trucker**
I always pick trucker but I feel like if a captain is realizing that they get destroyed often they will choose the other one. I believe it's a fair choice you have to make between getting a more active role in breaches or getting talents even faster.

**Trusted and Esteemed Captain**
These are the main reason to pick this tree first, and it does feel like talent progression would be far slower without them, they're quite fair overall and i don't think they need any changes.

**Shipmaster tree**
You usually get this tree first if you want to do nothing but support the crew.

**Commander**
This talent is fine, it just sets the ground for the next one with some minor buffs to whoever gets chosen as the captain's favorite son.

**Trust Building** and **Comradierie** vs **Maintenance Routine** and **Inspire to Battle**
Here the tree seems to be literally asking you to pick Maintenance Routine and Inspire to Battle since the first and fourth talents in this tree do nothing but wait for you to chose the 2 mentioned talents. But the other 2 talents give very noticeable rewards for efficiency at the job. I believe this is a fair choice and are needless of changes.

**Engineer**
The thing about the engineer is its ability to change the course of the game from normal to a fully depleted fuel/radiation based experience.

**Weapons Engineer tree**
**Military Applications**
Right away the tree gives you the ability to fabricate the second most versatile coilgun ammunition in the game, right behind physicorium. Which I consider to be a mildly insane considering how easy it is to find fuel and how quick it drains. Even tho it would leave the first talent to be a bit bland, I'd move the recipe for deplted fuel coilgun ammunition to a tier 2 or 3 talent.

**Arms Dealer** and **Strengthened alloys**
Here everyone i knew, know and will know picks arms dealer, the tools from strengthened alloys are a direct downgrade from the dementionite tools from the mechanic and most crews I've seen simply don't have a second engineer to pick both of these talents (unless you're playing singleplayer of course). Arms dealer gives you free resources in considerable amounts for every mission and gives you straight upgrades from normal ammunition kinds. The easiest solution to the problem is to give either a big buff to strengthened alloys talent, maybe make it so it gives you one or two depleted fuels every time you complete a mission.

**Castle Doctrine** and **Expanded Research**
Honestly these two talents are kind of just... lame?
Dirty bombs leaves a shitty cloud behind that is useless unless the enemy is standing perfectly still for 20 seconds, the extra damage against enemies inside the sub is ok tho.
Expanded research's free exp is cool but... it feels like you couldn't think of something better?
Both of these talents are fine and balanced in every way but man you could've done something a bit more fun, no?

**Nuclear Option**
The RFA is straight up the best or second best handheld weapon in the game doubtless, the rate of fire isn't half bad and the damage is insane, this is one of the if not the only weapon that can face to face fight an outpost security guard and win reliably. So any monster that doesn't come from the abyss is pretty much a joke.
Ammo isn't a problem because if fuel wasn't easy to get enough already now you can get 2 free fuel every mission and deconstruct one of those for another 2 free fuel!
and then there's the 100% nausea resistance which is the cherry on the top.
Suggestion: Nerf the RFA to the ground.

**Electrician**
**Just a Scratch** and **Minor in Mechanics**
They're a fair choice, self healing vs faster skill learning and a handheld status monitor. Really helpful early game.

**In The Flow** and **Melodic Raspite**
Another fair choice, either help yourself to repair or help others, I'd buff melodic raspite a bit tho, maybe make the guitar a toolbelt item.

**Electrochemist** and **Optimized Power-Flow**
Yet another fair choice, self defense and better batteries or better turret power management and a skill bonus, pretty balanced, i'd just cut the max skill bonus from Optimized Power-Flow in half.

**All-Seeing Eye**
Aaaaand... all the fair choices you've made are gone because now you have everything, cool, awesome.

**Physicist tree**
This is the tree that gives the crew a choice to either continue as normal or base their entire game around volatile fuel rods and pucs.

**Remote Monitor**
cool

**Co-Author** and **Hazardous Materials**
Both of these are pretty good talents, you can get your crew to max mech and elec skill super fast if you remember to write the books, and volatile fuel rods can last (literally) days.

**College Athletics** and **Polymath**
Both are fair

**Ph.D in Nuclear Physics**
Yeah yeah the pucs is good and all but i want to talk about how stupidly strong the gordon freeman playstyle is. If you take a dementionite crowbar and  you have this talent and college athletics you can kill late-game enemies like they're frickin' hammerhead spawns.
(and this is not a masterwork crowbar)
https://user-images.githubusercontent.com/55637865/159959312-7370089d-61d8-46a8-ad94-01827bb561d7.mp4

**Mechanic**
A lot of mechanic's talents are only useful when you're controlling it, making it a bit underwhelming in singleplayer if it wasn't for the technician tree.

**Machinist tree**
**Tinkerer**
You don't see people tinkering that often but it's pretty strong, being able to drain a submarine 4 times as fast is insane for some subs which only downside is their slow draining (humpback). For how ridiculous it sounds i think it's fair and mostly fun since it gives the mechanic a way more active role than just running around fixing leaks and fixing stuff.

**Mass Production** and **Quickfix**
Most people i know get mass production and it makes sense, crafting is boring and it doesn't have an option to auto-craft yet, so spending less time sticking to the fabricator is welcomed. Now quickfix is not half bad either, repairing is also boring for some people, now you can do it twice as fast so you can get back to clowning faster. Both of these talents are more qol features than anything, the balance of it doesn't really matter.

**Hull Sealer** and **Overcloking**
This is where the line between "fun" and "so unbalanced it's lame" is broken, the fixfoam grenade fix walls so fast  that you can have an entire team of mechanics in a room and they wouldn't be able to match it. And overclocking allows you to literally tank 4 pieces of hull being GONE and still having a dry room if you're tinkering a pump in it.

**Let It Drain**
The portable pump is actually a nice addition altough you don't see it used that often, people usually get it to drain rooms faster but you already have tinkering for that too, so it's a bit of a weird decision. You can empty wrecks and ruins with it I guess that's kinda cool. The other part of the talent is there I guess, as mentioned early you don't run into fuel problems often now.

**Technician**
**Aggressive Engineering** and **Safety First**
The first part of aggressive engineering is a bit underwhelming, mostly an escape tool than anything, the 20% past max condition is cool tho. Meanwhile safety first gives you the ability to progress on electrical skill faster while you repair. The safety harness is awesome until the room is filled with pressure. I think at some point i remember someone tried to make aggressive engineering being able to repair items by hitting them with the wrench, would've loved to see that, and would've made the talent probably worth getting over safety first.

**Hull Fixer**
Since the welding speed was decreased this talent is very welcome.

**Crisis Management** and **Pyromaniac**
Crisis management rewards the hull fixing mechanic playstyle while pyromaniac is the one and only talent that can turn the mechanic into a combat mechanic, I think it's a pretty fair choice but I wouldn't make the alien pistols work with pyromaniac, since it's already medic's thing.

**Artisan Smith**
These are just the late endgame tools every mechanic equip their crew with, not overpowered, but a great help.

**Pioneer tree**
**Scrounger**
Pretty fun, introduces the scrap guy playstyle, helps looting wrecks, but still weak enough to be a tier 1 talent.

**Demolitions Expert** and **Miner**
If railgun shells could have quality and demolitions expert affected those, it would be more viable, but right now most other explosives are only crafting materials or used as payload, grenades are affected by the talent but most of the times, the enemy either doesn't die from the grenade or dies from the grenade, there's never a situation where the enemy survives with little health that could've been drained by a masterwork grenade. Now miner is awesome, makes mining in caves and abyss islands less tedious and helps breaching walls, the extra ore output is awesome too, a bit too awesome, maybe reduce that chance.

**Canned Heat** and **Elbow Grease**
The accordion thing is fun on canned heat, the ability to make masterwork tanks is awesome too. But elbow grease allows your mechanic o get up to about 100 more hp and 100% more melee power if you get the right talents and gear, which is bonkers. You should probably add a cap to that.

**Scrap Savant**
There's better ways to find alien materials but this talent allows you to get exactly the material you want. Not overpowered, very original and fun.

**Medic**
The medic unlocks the most lategame equipment out of all of the classes

**Xenologist tree**
The power of the deconstructor

**Researcher's Intuition**
The first part of this talent is madness, what you should do is add a chance of finding extra alien materials, but not trinkets, because this talent mixed with the other talents that have to do with deconstructing artifacts gets absolutely insane. The second part is not useful since the artifact transport cases were added.

**Curiosity** and **Double Duty**
The amount of exp you get from curiosity doesn't make up for that juicy outdoor 25% physical damage reduction, so I recommend nerfing that reduction.

**Alien Hoarder** and **Reverse Engineer**
A medic with alien hoarder and combat medic can get an insane total dps that outmatches every other class in the game (mostly because the alien pistols by themselves are overpowered). Even then if the crew's medic is going to do nothing but loot ruins, the fractal guardian's burn protection is pretty damn high so if a player is going for that playstyle they'll choose reverse engineering which is also ridiculous and should be a 25% chance instead of 50%.

**Atmos Machine**
Who's idea was it to give players a chance of getting literally infinite endgame items

**Medic tree**
You're probably just getting this tree for the combat stimulant and the endocrine booster

**Field Medic**
Balanced

**Fireman's Carry** and **Mechanics Course**
These are both mildy useful in a lot of situations and I don't think they need a change

**Bedside Manner** and **Combat Medic**
Forget bedside manner let's talk about combat medic
PLEASE add a cap to how far the combat medic's buff can go, if the combat stimulant wasn't good enough now you can fire an autoshotgun like 65% faster and ram into walls at the speed of light, leaving your security guard in the corner crying without a job.

**Don't Die on Me!**
This one is nice, the CPR buff is noticeable and the endocrine boosters are the best lategame item in the game, I don't think you need to change anything because you rarely see any player get the 15 extra talents you can get from the endocrine boosters (or any endocrine booster) before they finish the campaign.

**Scientist tree**
**Dr. Submarine** and **Gene Therapist**
Dr. Submarine needs to be way stronger to make up for the 10% extra potency that gene therapist adds to all genes you combine, making powerful genes would be EXCRUCIATING if it wasn't for this talent.

**Research and Development** and **Research Grant**
That extra 45% exp is awesome for early game, plus the books do help getting to 100 med skill way faster. You can get an actually considerable amount of money from research grant, the pressure stabilizer is kind of useful in some situations too. I think it's a fair choice.

**Gene Harvester** and **Genetic Stability**
The problem here is that you can get to 0% chance of making a gene go bad by waiting to get "Advanced Splicing" to have a massive skill bonus on medical, which makes genetic stability quite useless. Plus gene harvester is one of the easiest ways to get huge amounts of genetical material, mostly with swarms of smaller enemies.

**Advanced Splicing**
Quite alright, players usually want to get this talent before actually starting to refine your unidentified genetical material.

**Security Officer**
Most of the choices you get to make here are between useful and fun, which I don't fully like but it seems like the community does.

**Specialist tree**
**First Aid Training** and **Physical Conditioning**
Medicine is not something that crews usually lack either, so getting first aid training just to not get 2 injections is not worth over the passive buffs physical conditioning gives.

**Navy Diver** and **Scavenger**
The slipsuit is awesome, the 25% physical damage resistance is insane and should be nerfed. Now scavenger is quite general and allows you to get even more loot from structures like wrecks and alien ruins since i believe medic's and mechanic's looting talents don't cancel scavenger's ability to spawn more loot. I think scavenger is fine but that 25% from the navy diver has to be reduced.

**Buccaneer** vs **Deep Sea Slayer**
Buccaneer's power attack is just not worth the trouble unless it's a far larger increase, something that allows you to (at least) 1shot a crawlerhusk with a masterwork axe even if you hit it's tail. Deep Sea Slayer is fine, it can even get a bit ridiculous with ahab's spears, and I don't think it needs the chance of giving out alien materials.

**Dual Spec-Ops**
The dash allows you to move faster than someone with a scooter, the resistance it gives you while dashing is not that noticeable tho.
The free points are nice.

**Warden**
**Enforcer**
Balanced

**Beat Cop** and **Munitions Expertise**
Beat cop is pretty fun and changes combat a bit for the better. Munitions Expertise benefits a less combat-ish playstyle but those exploding slugs are pretty good.

**By The Book** and **Stone Wall**
By The Book is really fun and rewards a change of pace, and simply stone wall makes you (more of) a tank.

**Implacable**
I see a lot of people arguing about the autoshotgun and I myself know it's overpowered, but i can't think of a way to nerf it that isn't ridiculous, I thought of making it a burst fire weapon with very long delays between bursts, maybe?.
The rest of the talent is fun, that feeling of power and anger when you should be dead is great.

**Soldier tree**
**Bootcamp**
Players usually get this one first so they get the other trees faster, I guess that is fine.

**Gunsmith** and **Warrior Poet**
Gunsmith allows you to make masterwork weapons, but if you're the one that sits in a turret and gets every enemy that comes near the sub you can get considerable amounts of money with warrior poet. I think it's a fair choice.

**Expert Commando** and **Tandem Fire**
The fact that you can crouch underwater is the only thing that makes expert commando not worse (yet not better) than than tandem fire. The bleeding recovery is also noticeable and a huge part of the talent, but the firepower you get from Tandem Fire is almost too good.

**Warlord**
I personally really like the assault rifle but most people prefer the other weapons because they are way better, which is true. The assault rifle has no penetration which i think is what it needs for it to shine as a weapon.
The 5% to deal twice the damage is barely noticeable, I think 7-10% would be better.

Overall security talents don't have much of an impact of the crew's game, they just make him a tank with cool guns

i haven't played assistant enough to give you my take on its talents but last i was in the discord server everyone said it was still overpowered

I know i didn't do much other than describe talents in this whole thing but I want to give the devs an idea of what's the current situation of a lot of these

Do upvote if you agree and feel free to add ideas and feedback in the comments

**Comment by ubertpendragon** [comment]
**"By the book"** is a great talent but it's tier 3 and it's really late in the tree for a talent that is all about XP.
It should be probably be T2, and **Munitions expertise** T3. 
The choices of **"By the book" vs "Beat cop" (T2)** and **"Stonewall" vs "Munitions Expertise" (T3)** feels more balanced

**Military Applications,  Strengthened alloys, Arms Dealer**: You can probably swap Strengthened alloy and Military applications, and put the depleted fuel rods on Strengthened alloys. 
T1 you get hardened tools, T2 you have to pick between depleted Coilgun ammo or depleted SMG/Revolver ammo.

The **xenologist** branch of medic absolutely breaks the economy. Ruins are already overloaded with alien materials, appear every cave, and you can double-triple dip with Xenologist

**Let It Drain**: The portable pump is a miniature black-hole. 1000 drain power, with a single battery. It's the equivalent of 10 small pumps, and almost 2 big pumps. Decrease the drain/power consumption so it's more of a support pump than a portal that sucks water into the void


But overall I think players will always tend to pick the talent that gives you a new crafting recipe/item over the other talent. 
No one likes being locked out of items, especially since you can equip the whole crew with it, and some of these are bonkers.

Cargo scooter, Slipsuit, Fixfoam and Depleted ammo come to mind.

Here are some adjustments:
-Cargo scooter: give **Polymath** the recipe too? It just feels bad to not have the scooter
-Clown diving mask: also give **True Potential** the recipe? (if you skipped **Water Prankster** you can still RP as clown when you reach T4)
-Depleted ammo: as above, make Depleted coilgun vs Depleted SMG/Revolver a choice
-Fixfoam: change the recipe to Sodium + Phosphorus + Graphite. Yes, _Graphite_, the raw mineral, so this makes the fixfoams pretty scarce.
-Slipsuit: increase the cost by adding 2 swim bladders to the recipe. Now you gotta earn your slipsuit

**Comment by Regalis11** [comment]
Thank you for taking the time to write this! We're planning on doing an overhaul/balance pass on the talents in the near future, and we'll be sure to take all this feedback into consideration. Linked this to a talent balancing ticket in our private repo https://github.com/Regalis11/Barotrauma-development/issues/3205

**Comment by itchyOwl** [comment]
Good feedback = constructive feedback. Thank you for it!
